frontend/API_COMPATIBILITY

```diff
@@ -0,0 +1 @@
+0.33.x
```

Comment:
@nastasi-oq hi, I follow up on the Skype message you sent me:

> Hi Thomas, about version consistency check I would propose req_major == be_major and req_minor <= be_minor , following the rules, major changes when the compatibility is broken and minor changes when compatible changes are added.

I'd start with something which requires less change. If minor version (here, `33`) does not break, we should not have to worry about much about it. If minor changes breaks compatibility, well, it should be a major change then.